src/gui/accountsettings.h

```diff
@@ -54,11 +54,16 @@ class AccountSettings : public QWidget
     Q_PROPERTY(AccountStatePtr accountState MEMBER _accountState)
 
 public:
+    enum class ModalWidgetSizePolicy { Minimum = QSizePolicy::Minimum, Expanding = QSizePolicy::Expanding };
```

Comment:
Do we really need both?

Reply:
Yes, see the screenshot, if we bloat the login dialog to full-screen it looks horrible.